src/gpu/intel/jit/gemm/gen_gemm.hpp

```diff
@@ -262,7 +262,8 @@ struct gen_gemm_t : public gpu_gemm_t {
                     ? attr_zps.get_data_type(swap_ab_ ? DNNL_ARG_A : DNNL_ARG_B)
                     : data_type::s32;
             bool int_acc = utils::one_of(eff_a_type(), s8, u8);
-            int_acc &= !a_scales_2d();
+            //int_acc &= (!a_scales_2d() && !b_scales_2d());
+            int_acc &= (!a_scales_2d() && !b_scales_2d());
```

Comment:
I am not following the logic here. Shouldn't we need to be checking against the scales data type?